tests/generation/test_utils.py

```diff
@@ -56,6 +56,8 @@
     import torch
     import torch.nn.functional as F
 
+    torch._dynamo.config.cache_size_limit = 512  # So we can compile all models with no cache limit exception
```

Comment:
a side question: do we (ever) call `torchdynamo.reset()` or similar method for compilation tests? Would that avoid the `cache limit exception` without set it to `512` here?

Reply:
We can reset the cache in dynamo instead of this, yes. Do you prefer it?

[The advantage of this method would be slightly faster compilation, but the difference should be very small]

Reply:
no, you can keep what it is. I am just thinking in more general: such stuff is always good to be cleaned after each test call (teardown). I have seen similar issue for torchscript and have to add `clear_torch_jit_class_registry` (otherwise memory leak)